PocketKit/Sources/PocketKit/PremiumUpgrade/Model/ReceiptService.swift

```diff
@@ -27,6 +27,9 @@ class AppStoreReceiptService: NSObject, ReceiptService {
     }
 
     func send(_ product: Product?) async throws {
+        // First make sure a receipt even exists before we try and get one.
+        _ = try getReceipt()
```

Comment:
wondering if, instead of calling `getReceipt()` twice, we can just handle the whole thing inside `getReceipt()` where we already check the existence on disk?